src/AidSelectionStep/AidSelectionStep.tsx

```diff
@@ -1,15 +1,41 @@
 import React, { FunctionComponent, useState } from 'react';
-import { Divider, Typography, Select, Alert } from 'antd';
+import { Typography, Select, Divider, Alert } from 'antd';
 import { InfoCircleOutlined } from '@ant-design/icons';
 
 import { NotebookNavAnchor, NotebookNavStep } from '../Notebook';
 import { TableSchema } from '../types';
 
+import { useMissingAid } from './use-missing-aid';
+
 import './AidSelectionStep.css';
 
 const { Title } = Typography;
 const { Option } = Select;
 
+type MissingAidWarningProps = {
+  schema: TableSchema;
+  aidColumn: string;
+};
+
+const MissingAidWarning: FunctionComponent<MissingAidWarningProps> = ({ schema, aidColumn }) => {
+  const computedResult = useMissingAid(schema, aidColumn);
+
+  return computedResult.state === 'completed' && computedResult.value ? (
+    <Alert
+      className="ColumnSelectionStep-notice"
```

Comment:
Stale class name. `MissingAidWarning` or maybe not needed at all?

Reply:
good catch, I'm still css-blind a bit. I'll reuse `"AidSelectionStep-notice"`, I guess style-wise these two notices don't differ

Reply:
We have this in-house CSS rule where we name classes based on components. If it's the root rendered element then it's just `ComponentName`. If it's some sub-element then we use `ComponentName-part`.

Reply:
The current component is `MissingAidWarning` so that should be the class name.